delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/ElasticSearchViewRejection.scala

```diff
@@ -32,21 +36,24 @@ object ElasticSearchViewRejection {
     *
     * @param tag the provided tag
     */
-  final case class TagNotFound(tag: Label) extends ElasticSearchViewRejection(s"Tag requested '$tag' not found.")
+  final case class TagNotFound(tag: Label, id: Iri, project: ProjectRef)
```

Comment:
Let's keep consistency. All the other resource types have the following fields
```scala
  final case class TagNotFound(tag: Label) extends ResolverRejection(s"Tag requested '$tag' not found.")
```

So either you keep it here in the same way or change it in all the others.